packages/reakit/src/Toolbar/ToolbarFocusable.tsx

```diff
@@ -145,10 +166,20 @@ class Component extends React.Component {
   }
 }
 
-hoistNonReactStatics(Component, Box);
+// @ts-ignore
+hoistNonReactStatics(ToolbarFocusableComponent, Box);
 
-const ToolbarFocusable = styled(Component)`
+const ToolbarFocusableStyled = styled(ToolbarFocusableComponent)`
```

Comment:
That's the name that will appear on devtools, so I think we should keep it as `ToolbarFocusable`.

Reply:
`ToolbarFocusableComponent` is a good idea though. Maybe we should adopt it as a pattern for other components too. :)